docs/device_api.md

```diff
@@ -332,7 +332,7 @@ MQTT topic:
 ```
 <apiKey>/<deviceId>/cmd
 ```
-where it will receive the command information.
+where it will receive the command information. Please note that the device should subscribe to the broker using the --disable-clean-session option. This option means that all of the subscriptions for the device will be maintained after it disconnects, along with subsequent QoS 1 and QoS 2 commands that arrive. When the device reconnects, it will receive all of the queued commands.
```

Comment:
I understand that `--disable-clean-session` is a falg in mosquitto_sub. However, the device could be using (most probably) another software to connect. Maybe we should mention the "neutral name" of the option in MQTT protocol, then using `--disable-clean-session` as an example. Something like this:

"Please note that the device should subscribe to the broker using the disabled clean session mode (enabled using `--disable-clean-session option` CLI parameter in mosquitto_sub)"

(No sure if "disabled clean session mode" is the right neutral name or shoud be other)

Reply:
Agree